python-hawkular-client.spec

```diff
@@ -0,0 +1,114 @@
+%global srcname hawkular-client
+%global _docdir_fmt %{name}
+
+%if 0%{?fedora}
+%bcond_without python3
+%else
+%bcond_with python3
+%endif
+
+Name:           python-%{srcname}
+Version:        0.5.1
+Release:        1%{?dist}
+Summary:        Python client to communicate with Hawkular server over HTTP(S)
+
+License:        ASL 2.0
+URL:            https://github.com/google/%{srcname}
```

Comment:
@zgalor what is this ? this is the wrong home page.
should be https://github.com/hawkular/%{srcname}-python